pkg/controller/operands/kubevirt.go

```diff
@@ -291,7 +292,7 @@ func (h *kvConfigHooks) addEnabledFeatureGates(req *common.HcoRequest, foundFgSp
 	return resultFg, fgChanged
 }
 
-func (h *kvConfigHooks) filterDisabledFeatureGates(req *common.HcoRequest, foundFgSplit []string, resultFg []string) (bool, []string) {
+func filterOutDisabledFeatureGates(req *common.HcoRequest, foundFgSplit []string, resultFg []string) (bool, []string) {
```

Comment:
why?

Reply:
I removed this function receiver because its not being used and this function is not exported.
So basically this function acts like any other "private" function you would create for keeping the code 
readable.